.github/workflows/setuptools.yml

```diff
@@ -5,14 +5,16 @@ on:
     branches: [ develop ]
   pull_request:
     branches: [ develop ]
+  schedule:
+    - cron: "0 8 * * *"
```

Comment:
Every day at 8AM? 

OK. Why, not.

Reply:
One day a week is enough? On Monday?

Reply:
Don't know. Let's try every day, and adjust later if needed.